sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -83,24 +83,25 @@ def read_porttab_mappings(self, porttabfile, asic_inst=0):
             # Mapping of logical port names available on a system to ASIC instance
             self.logical_to_asic[intf_name] = asic_inst
 
-        self.logical = logical
-        self.logical_to_physical = logical_to_physical
-        self.physical_to_logical = physical_to_logical
+        self.logical.extend(logical)
+        self.logical = list(set(self.logical))
```

Comment:
Do we need this? as "logical" is anyways a list which we extend and its elements should be unique

Reply:
The current logic of existing multiasic caller function read_all_porttab_mappings()  could parses the platform.json file number of times based on the number of asic.   The "logical" is not  unique.  and the 'extend' doesn't take care the duplicate element.

Reply:
@judyjoseph  - reminder of this.